core/services/job/models.go

```diff
@@ -66,7 +66,7 @@ var (
 		FluxMonitor:        true,
 		OffchainReporting:  false, // bootstrap jobs do not require it
 		OffchainReporting2: false, // bootstrap jobs do not require it
-		Keeper:             true,
+		Keeper:             false, // observationSource is injected when the job is validated
```

Comment:
I believe this comment is out of date now. the observation source is injected in the upkeep executor

Reply:
True, updating it.